android/src/main/java/com/swmansion/rnscreens/SearchBarManager.kt

```diff
@@ -48,17 +61,17 @@ class SearchBarManager : ViewGroupManager<SearchBarView>() {
     }
 
     @ReactProp(name = "barTintColor", customType = "Color")
-    fun setTintColor(view: SearchBarView, color: Int?) {
+    override fun setBarTintColor(view: SearchBarView, color: Int?) {
         view.tintColor = color
     }
 
     @ReactProp(name = "disableBackButtonOverride")
-    fun setDisableBackButtonOverride(view: SearchBarView, disableBackButtonOverride: Boolean?) {
-        view.shouldOverrideBackButton = disableBackButtonOverride != true
+    override fun setDisableBackButtonOverride(view: SearchBarView, value: Boolean) {
+        view.shouldOverrideBackButton = value != true
     }
```

Comment:
```suggestion
    override fun setDisableBackButtonOverride(view: SearchBarView, disableBackButtonOverride: Boolean) {
        view.shouldOverrideBackButton = disableBackButtonOverride != true
    }
```

Same here

Reply:
Especially that it does not remain unused.

Reply:
Changed in 3472960cb38ea20a6a963c47ec1fc76f0d8e4fd7